ts/Dashboards/EditMode/SidebarPopup.ts

```diff
@@ -162,12 +186,12 @@ class SidebarPopup extends BaseForm {
                 dropContext: Cell | Row
             ): Cell|void {
                 if (sidebar && dropContext) {
-                    return sidebar.onDropNewComponent(dropContext, {
+                    const options = {
                         cell: '',
                         type: 'KPI',
-                        title: 'Example KPI',
-                        value: 70
-                    });
+                        title: 'Example KPI'
+                    } as Partial<KPIComponent.ComponentOptions>;
```

Comment:
It would be better if you have full type check via
`const options: Partial<KPIComponent.ComponentOptions> =...`
than to force the type with fewer type checks
`... as Partial<KPIComponent.ComponentOptions>`.

Reply:
Set type implicited, is better than casting, I agree